packages/client/src/types.ts

```diff
@@ -36,6 +36,8 @@ export interface WorkflowExecutionInfo {
   taskQueue: string;
   status: { code: proto.temporal.api.enums.v1.WorkflowExecutionStatus; name: WorkflowExecutionStatusName };
   historyLength: number;
+  // Only available in server version 1.20 or later
```

Comment:
We want that comment to be visible to users. `// ...` comments are not.

Use a `/** ... */` block comment, and copy what you have on the `historySize` property in WorkflowInfo.

Reply:
Done